css/css-ui-3/text-overflow-016.html

```diff
@@ -0,0 +1,46 @@
+﻿<!DOCTYPE html>
+<meta charset="utf-8">
+<title>CSS Basic User Interface Test: clipping the ellipsis</title>
+<link rel="author" title="Florian Rivoal" href="http://florian.rivoal.net/">
+<link rel="help" href="http://www.w3.org/TR/css3-ui/#ellipsing-details">
+<link rel="match" href="../reference/ref-filled-green-100px-square.xht">
+<meta name="flags" content="ahem">
+<meta name="assert" content="If there is insufficient space for the ellipsis, then clip the rendering of the ellipsis itself">
+<style>
+.test {
+  overflow: hidden;
+  text-overflow: ellipsis;
+  width: 100px;
+  white-space: pre;
+  line-height: 1;
+  font-size: 50px;
+  font-family: ahem;
+  color: green;
+}
+span {
+	display: inline-block;
+	width: 60px;
+	height: 1em;
+	vertical-align: bottom;
+	background: green;
+}
+.red {
+  position: absolute;
+  z-index: -1;
+  background: red;
+  width: 100px;
+  height: 50px;
+}
+.green {
+  position: absolute;
+  z-index: -2;
+  background: green;
+  width: 100px;
+  height: 100px;
+}
+</style>
+
+<p>Test passes if there is a filled green square and <strong>no red</strong>.</p>
+<div class=red></div>
+<div class=green></div>
+<div class=test><span></span>filler text to make things overflow</div>
```

Comment:
This doesn't test whether the ellipsis is clipped. It could be simply shifted to the left so that it overlaps the inline block.